Freshworks-Samples/App-Development-Features/Advanced-Features/request_method/request_api/app/app.js

```diff
@@ -1,16 +1,15 @@
-$(document).ready( function() {
-  app.initialized().then(function(client) {
-    client.events.on('app.activated', function() {
-      // sample code for RequestAPI 
-      client.request.get('https://httpbin.org/get', { client : true })
-        .then(
-          function(data) {
-            console.log('RequestAPI:', data);
-          }, 
-          function(error) {
-            console.log('RequestAPI:', error);
-          }
-        );
+$(document).ready(function () {
+  app.initialized().then(function (client) {
+    client.events.on("app.activated", function () {
+      // sample code for RequestAPI
+      client.request.get("https://httpbin.org/get", { client: true }).then(
+        function (data) {
+          console.log("RequestAPI:", data);
```

Comment:
```suggestion
          console.info("Success: The RequestAPI was successful");
          console.info(data)
```
Combining a string and an object is not a good idea and the result may not be as expected always.

Reply:
Please ignore commits from `request_api` app codebase. I will make changes to it eventually in a separate PR. 😄

Reply:
Could you please remove those commits out of this PR? :)

Reply:
Fixed this.